imports/server/announcements.ts

```diff
@@ -1,37 +1,11 @@
 import { check } from 'meteor/check';
 import { Meteor } from 'meteor/meteor';
-import Ansible from '../Ansible';
 import Announcements from '../lib/models/Announcements';
 import MeteorUsers from '../lib/models/MeteorUsers';
 import PendingAnnouncements from '../lib/models/PendingAnnouncements';
-import { userMayAddAnnouncementToHunt } from '../lib/permission_stubs';
 import JoinPublisher from './JoinPublisher';
 
 Meteor.methods({
-  postAnnouncement(huntId: unknown, message: unknown) {
```

Comment:
If I'm reading this right, the server bundle never imports `imports/methods/announcements/post`, and thus no method is defined on the server, which seems wrong?  Am I missing something, or is this PR missing a file or an import somewhere, or does this just not work as written?

Reply:
Yeah that's a mistake and I'm missing the import.